hles/transformation/src/main/scala/org/broadinstitute/monster/dap/DogTransformations.scala

```diff
@@ -6,24 +6,35 @@ import org.broadinstitute.monster.dogaging.jadeschema.table.HlesDog
 object DogTransformations {
 
   /** Parse all dog-related fields out of a raw RedCap record. */
-  def mapDog(rawRecord: RawRecord): HlesDog =
-    HlesDog(
-      dogId = rawRecord.id,
-      ownerId = rawRecord.getRequired("st_owner_id").toLong,
-      hlesDogStudyStatus = Some(StudyStatusTransformations.mapStudyStatus(rawRecord)),
-      hlesDogDemographics = Some(DemographicsTransformations.mapDemographics(rawRecord)),
-      hlesDogResidences = Some(DogResidenceTransformations.mapDogResidences(rawRecord)),
-      hlesDogPhysicalActivity =
-        Some(PhysicalActivityTransformations.mapPhysicalActivity(rawRecord)),
-      hlesDogResidentialEnvironment =
-        Some(ResidentialEnvironmentTransformations.mapResidentialEnvironment(rawRecord)),
-      hlesDogRoutineEnvironment =
-        Some(RoutineEnvironmentTransformations.mapRoutineEnvironment(rawRecord)),
-      hlesDogBehavior = Some(BehaviorTransformations.mapBehavior(rawRecord)),
-      hlesDogDiet = Some(DietTransformations.mapDiet(rawRecord)),
-      hlesDogMedsPreventatives =
-        Some(MedsAndPreventativesTransformations.mapMedsPreventatives(rawRecord)),
-      hlesDogHealthSummary = Some(HealthStatusTransformations.mapHealthSummary(rawRecord)),
-      hlesDogFutureStudies = Some(AdditionalStudiesTransformations.mapFutureStudies(rawRecord))
-    )
+  def mapDog(rawRecord: RawRecord): Option[HlesDog] = {
+    val dog_id = rawRecord.id
+
+    rawRecord.getOptional("st_owner_id") match {
+      case None =>
+        MissingOwnerId(s"Record $dog_id has more/less than 1 value for field st_owner_id")
```

Comment:
Should this be
```suggestion
        MissingOwnerId(s"Record $dog_id has more/less than 1 value for field st_owner_id").log
```

I don't see how `log` is called otherwise.

Reply:
It's odd, it looks like you added `log` and then removed it. Was it not needed?

Reply:
@quazi-broad you need to import `import org.broadinstitute.monster.dap.HLESurveyTransformationPipelineBuilder.logger` from the main builder so you have the implicit logger, and then call `.log` on `MissingOwnerId`. I'd also advise renaming `MissingOwnerId` to more explicitly be an error, but that's more my preference/opinion.

Reply:
Thank you Phil and Raaid, with the logger from the builder explicitly imported, I am able to call .log on the error now and don't get compile errors (that's why I had added and then removed it prior).